### ISSUE TITLE
Delivered but not sent to device (IOS)

### ISSUE THREAD

**Issue by psaravanan** [rpush/rpush#333]
I can able to sent Push notification to the device when IOS app is generated for development and installed on IOS devices.
But Push notification not sending to any IOS devices but Notification says its delivered when IOS app is generated for App store and installed on devices.

I don't know where to check the issue since notification record says its deliver and I also tried changing certificates. Kindly let me know what I am doing wrong here.
 
I am using 
Rpush 2.7.0
Rails 4.2.6
Ruby 2.2.5

Notification record:
#<Rpush::Client::ActiveRecord::Apns::Notification:0x00000006b57998
 id: xx,
 badge: 2,
 device_token: "xxxxxxxxxxxxxxx",
 sound: "default",
 alert: "Test msg",
 data: "{\"message\":\"Hello world\"}",
 expiry: 86400,
 delivered: true,
 delivered_at: Thu, 26 Jan 2017 19:36:05 UTC +00:00,
 failed: false,
 failed_at: nil,
 error_code: nil,
 error_description: nil,
 deliver_after: nil,
 created_at: Thu, 26 Jan 2017 19:35:36 UTC +00:00,
 updated_at: Thu, 26 Jan 2017 19:35:36 UTC +00:00,
 alert_is_json: false,
 type: "Rpush::Client::ActiveRecord::Apns::Notification",
 collapse_key: nil,
 delay_while_idle: false,
 registration_ids: nil,
 app_id: 7,
 retries: 0,
 uri: nil,
 fail_after: nil,
 processing: false,
 priority: nil,
 url_args: nil,
 category: nil,
 content_available: false,
 notification: nil>

**Comment by Kris-Simpson** [comment]
+1 have the same issue

**Comment by raghvendra** [comment]
Yes, I too get the issue. If I reset the delivered to false and delivered_at to null, then it sends it.

**Comment by lluis-sancho** [comment]
We have the same issue.

**Comment by Ritesh-Katare** [comment]
I am having slightly different issue,  Delivered but not received at device (iOS) this happens sometimes, sometime push notification get received at iOS.

**Comment by DanielJunior** [comment]
@psaravanan, your registration_ids is nil. May be this?
